skins/laika/src/scss/overrides.scss

```diff
@@ -6,6 +6,14 @@
 	margin-top: 30px;
 }
 
+.input {
+	max-width: 300px;
+}
+
+.is-danger, .is-warning {
+	font-size: 1em;
```

Comment:
Is there a requirement to have different font size for error messages? :thinking:

Reply:
Error messages in have a bigger font size by default, it looks pretty horrible in our design so I have to force it back to 1em (which is the default font-size of 14px).

Reply:
My concern was that we are overriding font size specifically for this. I did a search in `\laika` for `font-size`: 
Generally the font is `14px`, but we have here 1em, and for radio we have `13px`. The text in the input fields is set to font size `1rem`.
Overall we have different font sizes and different unit measurements. It can easily get out of hand. Also just remembered that Volker mentioned pixels are not accessibility friendly

Reply:
I generally only use em or rem. The only places where you should see pixels on font sizes is the root HTML element, if you see it anywhere else, please do point it out in a review.

The radio button labels still have 14px (or rather 1 em) but I have forced the actual round radio element to be 13 pixels to resolve the "googly eyes" problem for the most part. I would call that somewhat of a "hack" but it was the best I could do without completely rewriting the Buefy radio buttons, it has something to do with how the radio buttons get rendered when their reference font size is 14px.